src/NavItem.js

```diff
@@ -28,12 +28,8 @@ class NavItem extends React.Component {
   }
 
   handleClick(e) {
-    if (this.props.onSelect) {
-      e.preventDefault();
```

Comment:
we probably still want to do `e.preventDefault()` if `disabled` is set?

Reply:
Yes

Reply:
ok